boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -139,13 +139,16 @@
 	status = "okay";
 
 	mx25lm51245: ospi-nor-flash@0 {
-		compatible = "st,stm32-ospi-nor";
+		compatible = "jedec,ospi-nor";
 		label = "MX25LM51245";
 		reg = <0>;
-		ospi-max-frequency = <DT_FREQ_M(50)>;
+		spi-max-frequency = <DT_FREQ_M(50)>;
```

Comment:
The Example does not match the `properties:` `spi-max-frequency`

On a side note, is not the former`ospi-max-frequency` sounds better?